change/@ni-nimble-components-33b8266d-c20f-4ec8-92a9-a6ecf81fe328.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Changed Rich Text Viewer markdown property to an attribute",
+  "packageName": "@ni/nimble-components",
+  "email": "107654185+joseahdz@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
- Can you update the chat conversation stories to use the rich-text-viewer attribute binding instead of property binding. I think that will make the `show code` view clearer. Don't need to update the matrix tests.